src/aiidalab_qe/plugins/pdos/workchain.py

```diff
@@ -86,6 +87,8 @@ def get_builder(codes, structure, parameters, **kwargs):
             overrides=overrides,
             **kwargs,
         )
+        # include nbands_factor (Same as in BandsWorkChain)
+        pdos["nbands_factor"] = orm.Float(3.0)
```

Comment:
Suggest to read the value directly from the `PwBandsWorkChain` so that the value always keeps updated with the bands.
```suggestion
        pdos["nbands_factor"] = orm.Float(PwBandsWorkChain.get_protocol_inputs()['nbands_factor'])
```